diff --git a/node_modules/passport-openidconnect/lib/profile.js b/node_modules/passport-openidconnect/lib/profile.js
index eeabf4e..8abe391 100644
--- a/node_modules/passport-openidconnect/lib/profile.js
+++ b/node_modules/passport-openidconnect/lib/profile.js
@@ -17,6 +17,7 @@ exports.parse = function(json) {
     if (json.middle_name) { profile.name.middleName = json.middle_name; }
   }
   if (json.email) { profile.emails = [ { value: json.email } ]; }
+  if (json.groups) { profile.groups = [ { value: json.groups } ]; }
   
   return profile;
 };
